receptor_affinity/mesh.py

```diff
@@ -427,6 +418,30 @@ def ping(self, count=10):
             results[node.name] = node.ping(count)
         return results
 
+    def settle(self):
```

Comment:
The implementation of `settle` looks good to me

Reply:
Cool. Thanks for checking!